index.php

```diff
@@ -271,7 +271,8 @@
 </li>
 <li value=106> <a href="extensions/EXT/EGL_EXT_pixel_format_float.txt">EGL_EXT_pixel_format_float</a>
 </li>
-<li value=107> <a href="extensions/EXT/EGL_EXT_gl_colorspace_bt2020_linear.txt">EGL_EXT_gl_colorspace_bt2020_linear</a>
+<li value=107> <a href="extensions/EXT/EGL_EXT_gl_colorspace_bt2020_linear.txt">EGL_EXT_gl_colorspace_bt2020_hlg</a>
```

Comment:
This is generated from registry.tcl, so after merging I'll tweak that to be functionally equivalent.